.github/workflows/ruby_tests.yml

```
@@ -37,7 +37,7 @@ jobs:
       fail-fast: true
       matrix:
         foreman-repo: [theforeman/foreman]
-        foreman-core-branch: [develop]
+        foreman-core-branch: [3.3-stable]
```

What do you think about testing both compatible versions of Foreman (3.3 and 3.4)?

```suggestion
        foreman-core-branch: [3.3-stable, 3.4-stable]
```

---

Sounds good, I will add it

---

the test failed for foreman 3.4 stable due to the graphql issue.
Is it safe to cherry-pick #322 into our foreman_puppet 4.0-stable branch which also goes with foreman 3.3?
or should we split the branch at this point into:
4.0-stable => foreman 3.3
4.1-stable => foreman 3.4

what do you think @Manisha15 @kamils-iRonin?

---

I opened PR against `4.0-stable` branch https://github.com/theforeman/foreman_puppet/pull/324